packages/compass-e2e-tests/helpers/read-data.ts

```diff
@@ -0,0 +1,29 @@
+import chai from 'chai';
```

Comment:
read-data is a very generic name – since this appears to be about our Document components, the filename could maybe reflect that it’s about reading document contents?